maap/maap.py

```diff
@@ -272,25 +272,27 @@ def listJobs(self, username=None,
         Returns a list of jobs for a given user that matches query params provided.
 
         Args:
-            username (str): Platform user.
-            algo_id (str): Algorithm type.
+            username (str, optional): Platform user. If no username is provided, the profile username will be used.
+            algo_id (str, optional): Algorithm type.
             end_time (str, optional): Specifying this parameter will return all jobs that have completed from the provided end time to now. e.g. 2024-01-01 or 2024-01-01T00:00:00.000000Z.
             get_job_details (bool, optional): Flag that determines whether to return a detailed job list or a compact list containing just the job ids and their associated job tags. Default is True.
-            offset (int, optional): Offset for pagination.
-            page_size (int, optional): Page size for pagination.
-            priority (int, optional): Job processing priority. Valid values are integers from 0-9.
+            offset (int, optional): Offset for pagination. Default is 0.
+            page_size (int, optional): Page size for pagination. Default is 10.
             queue (str, optional): Job processing resource.
             start_time (str, optional): Specifying this parameter will return all jobs that have started from the provided start time to now. e.g. 2024-01-01 or 2024-01-01T00:00:00.000000Z.
-            status (str, optional): Job status e.g. job-completed, job-failed, job-started, job-queued.
-            tag (str, optional): User job tag.
-            version (str, optional): Algorithm version i.e. GitHub branch.
+            status (str, optional): Job status, e.g. job-completed, job-failed, job-started, job-queued.
+            tag (str, optional): User job tag/identifier.
+            version (str, optional): Algorithm version, e.g. GitHub branch or tag.
 
         Returns:
             list: List of jobs for a given user that matches query params provided.
         """
         if username is None and self.profile is not None and 'username' in self.profile.account_info().keys():
             username = self.profile.account_info()['username']
 
+        if username is None:
+            raise ValueError("Username must be supplied.")
```

Comment:
Since the username field is optional, I think this error should be more descriptive. 
Something like, `Unable to determine username from profile, please provide a username`

Thoughts:
In the future we would need to add a method to explicitly do maap.login() by providing the PGT token if not found in the env vars, and then update this error message to be `Unable to determine username from profile, please ensured you are logged in by calling maap.login()` cc @bsatoriu